refarch-integrations/refarch-email-integration/refarch-email-integration-rest-example/src/main/resources/application-local.yml

```diff
@@ -6,6 +6,20 @@ spring:
     port: 1025
     username: test@muenchen.de
     password: secret
+  security:
+    oauth2:
+      client:
+        provider:
+          sso:
+            issuer-uri: http://keycloak:8100/auth/realms/local_realm
+            user-info-uri: ${spring.security.oauth2.client.provider.keycloak.issuer-uri}/protocol/openid-connect/userinfo
```

Comment:
Where is `${spring.security.oauth2.client.provider.keycloak.issuer-uri}` coming from?

Reply:
Fixed, i forgot to rename that when renaming the parent key